### ISSUE TITLE
.firebaserc project config property 

### ISSUE THREAD

**Issue by will-hill** [clairechabas/auth-firebase-next-react-jotai#2]
Hi Claire, this is the best Next + Firebase Auth content I have come across and I have read a lot so far.  I am reading this repo and also your related Medium article https://medium.com/gitconnected/auth-with-firebase-for-your-next-js-react-app-in-2023-a30b65b5b2b1.   
  
One issue I ran into is the project config in the .firebaserc file.  I figured it out, replaced your project id with mine.     
  
I thought you might want to update your doc or something.

**Comment by Dranaxel** [comment]
I agree on both: great boilerplates and it lack a line in the readme saying to change this line.
